README.md

```diff
@@ -35,6 +35,28 @@ If you have a recipe that you'd like to add, either open a PR or leave a comment
 
 - Smoothies made with pea protein powder: plant milk, chia seeds, and fruit. Keep half a dozen kinds of frozen fruit chunks in your freezer and let your kids pick what they want. Add nut butter too, if your kids are not allergic to it.
 
+## Breakfast Bars
+Easy to make breakfast bars that can be eaten cold or warmed up.
+
+### Ingredients
+- 2 cups old fashioned oats
+- 1.5 cups whole wheat flour
+- 1/2 cup toasted wheat germ
+- 2 tsp cinnamon
+- 1/2 tsp salt
+- 1 cup honey
+- 2/3 cup unsweetened apple sauce (we use two of the individual serving cups)
+- 1/2 cup canola oil
```

Comment:
I wonder if this would work with a different kind of oil such as coconut oil. I've heard a lot of [ugly things about canola oil](https://www.medicalnewstoday.com/articles/canola#risks) lately.  

![IMG_0065](https://github.com/bithippie/good-energy-kid-breakfast/assets/70786/07465c31-9efc-43ad-8e56-2ba70342a33e)

Reply:
I think as long as you're using a neutral oil it should be fine. We tried with olive oil once in a pinch and it tasted a little off.